cmd/config.go

```diff
@@ -34,8 +34,12 @@ func addConfigFlags(flags *pflag.FlagSet) {
 	flags.String("shell", "", "shell command to which other commands should be appended")
 
 	flags.String("auth.method", string(auth.MethodJSONAuth), "authentication type")
-	flags.String("auth.header", "", "HTTP header for auth.method=proxy")
+	flags.String("auth.header", "", "HTTP header for auth.method=proxy and auth.method=jwt-header")
 	flags.String("auth.command", "", "command for auth.method=hook")
+	flags.String("auth.aud", "", "The Application Audience (AUD) tag for JWT validation auth.method=jwt-header")
+	flags.String("auth.iss", "", "The Issuer (AUD) for JWT validation auth.method=jwt-header")
+	flags.String("auth.certsurl", "", "The URL to download certs from for JWT validation auth.method=jwt-header")
+	flags.String("auth.claim", "", "The claim which will contain the username auth.method=jwt-header")
```

Comment:
I think adding a namespace would be better. IE auth.jwt-header.header, auth.jwt-header.aud, auth.jwt-header.iss ...